Makefile

```diff
@@ -0,0 +1,46 @@
+SHELL=/usr/bin/env bash -o pipefail
+
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/observatorium/observatorium-operator:latest
+BIN_DIR ?= $(shell pwd)/tmp/bin
+
+CONTROLLER_GEN ?= $(BIN_DIR)/controller-gen
+
+# Install CRDs into a cluster
+install: manifests
+	kubectl apply -f deploy/crds
```

Comment:
Do we actually use this anywhere in the stack? I don't think it's in the tests, no?

Reply:
Removed